cmd/vicadmin/logs.go

```diff
@@ -276,9 +276,9 @@ func tarEntries(readers map[string]entryReader, out io.Writer) error {
 
 		// be explicit about the number of bytes to copy as the log files will likely
 		// be written to during this exercise
-		_, err = io.CopyN(t, e, sz)
 		if e != nil {
 			e.Close()
+			_, err = io.CopyN(t, e, sz)
```

Comment:
Interesting, I didn't see George's comment while I was making mine even though he left it earlier.